plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/MachineExtension.java

```diff
@@ -82,6 +84,16 @@
 
     private final PerspectiveManager perspectiveManager;
 
+    /**
+     * Controls central toolbar action group visibility. Use for example next snippet:
+     * <code>
+     *     bindConstant().annotatedWith(Names.named("machine.extension.central.toolbar.visibility")).to(false);
+     * </code>
+     * to define a constant. If no constant defined than default value is used - <code>true</code>.
+     */
+    @Inject(optional = true)
+    @Named("machine.extension.central.toolbar.visibility") boolean centralToolbarVisible = true;
```

Comment:
Lets name it "central.toolbar.visibility" without "machine.extension" 
and try to define it like constant in IdeActions